Makefile

```diff
@@ -165,6 +165,10 @@ $(VM_IMAGE): $(VM_DEP) bots
 	rm -f $(VM_IMAGE) $(VM_IMAGE).qcow2
 	bots/image-customize -v $(VM_PACKAGE) -s $(CURDIR)/test/vm.install $(TEST_OS)
 	$(RAWHIDE)
+	# HACK: systemd kills the services after 90s
+	# See https://github.com/containers/podman/issues/8751
+	bots/image-customize -r "sed -i 's/Type=notify/Type=exec/' /usr/lib/systemd/system/podman.service" -r "sed -i 's/Type=notify/Type=exec/' /usr/lib/systemd/user/podman.service" $(TEST_OS)
```

Comment:
I asked above to put this into the `RAWHIDE=` definition, but indeed you said in the upstream report that it already landed in F32/33, and Debian also has 2.2.1 now.